modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,29 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+== New features {SMProductName} version 2.4.4
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.10 and later versions.
```

Comment:
Should this be 4.11 and later versions?

Reply:
According to https://access.redhat.com/support/policy/updates/openshift#ossm , 4.10 is supported until Release of 2.5 + 1 month.

Reply:
Updated to reflect 4.11